Notes/NewNoteViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  NewNoteViewController.swift
+//  Notes
+//
+//  Created by Kathy Nguyen on 2/22/21.
+//
+
+import UIKit
+
+class NewNoteViewController: UIViewController {
+    @IBOutlet var titleField: UITextField!
+    @IBOutlet var noteField: UITextView!
+    
+    // used to get data from this controller
+    public var completion: ((String, String) -> Void)?
```

Comment:
The Callback function can be defined as typealiase. You can probably use the Result object?